training/models.py

```diff
@@ -86,6 +86,7 @@ class Participant(models.Model):
 	reg_approval_status = models.PositiveSmallIntegerField(default=0)
 	company = models.ForeignKey(Company, on_delete=models.PROTECT, null=True, blank=True)
 	city = models.ForeignKey(City, on_delete=models.PROTECT, null=True, blank=True)
+  source = models.CharField(max_length=25, null=True, default=None)
```

Comment:
The indentation looks incorrect. Is this some Space vs Tab issue? If yes, replace Tab by 4 Spaces in the entire file.

Reply:
Resolved